web-app/css/openboxes.css

```diff
@@ -2198,17 +2198,24 @@ img.photo_tiny {
 
 }
 
+.banner-menu,
+.banner-menu button,
+.banner-menu input {
+    height: 30px;
```

Comment:
this little bit of CSS properly aligns the search box and buttons on the banner.